src/main/java/com/entropyteam/entropay/employees/common/BaseRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.entropyteam.entropay.employees.common;
```

Comment:
I'd move this interface to something like com.entropyteam.entropay.common.repository, in order to keep it outside employees or any other module-specific package.

Reply:
ideally should be a common artifact, but we can do a common package for now